packages/back-end/src/routers/fact-table/fact-table.controller.ts

```diff
@@ -331,19 +322,13 @@ export const getFactMetrics = async (
 };
 
 export const postFactMetric = async (
-  req: AuthRequest<CreateFactMetricProps>,
+  req: AuthRequest<unknown>,
```

Comment:
Why do we lose this (`CreateFactMetricProps`)? Couldn't it be obtained from the FactMetricModel class?

Reply:
`unknown` is more accurate here since this is raw input data coming directly from users.  We would need to validate it first before knowing that it conforms to the type we expect.  The data model class does this validation already, so everything works fine now.  Changing the type here just prevents people making mistakes in the future and assuming something is validated when it's not.